sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1376,7 +1376,19 @@ aggregates AS (
         ORDER BY
           submission_timestamp
       )
-    ) AS os_environment_is_taskbar_pinned_private
+    ) AS os_environment_is_taskbar_pinned_private,
+    SUM(bookmark_migrations_quantity_chrome) AS bookmark_migrations_quantity_chrome,
+    SUM(bookmark_migrations_quantity_edge) AS bookmark_migrations_quantity_edge,
+    SUM(bookmark_migrations_quantity_safari) AS bookmark_migrations_quantity_safari,
+    SUM(bookmark_migrations_quantity_all) AS bookmark_migrations_quantity_all,
+    SUM(history_migrations_quantity_chrome) AS history_migrations_quantity_chrome,
+    SUM(history_migrations_quantity_edge) AS history_migrations_quantity_edge,
+    SUM(history_migrations_quantity_safari) AS history_migrations_quantity_safari,
+    SUM(history_migrations_quantity_all) AS history_migrations_quantity_all,
+    SUM(logins_migrations_quantity_chrome) AS logins_migrations_quantity_chrome,
+    SUM(logins_migrations_quantity_edge) AS logins_migrations_quantity_edge,
+    SUM(logins_migrations_quantity_safari) AS logins_migrations_quantity_safari,
+    SUM(logins_migrations_quantity_all) AS logins_migrations_quantity_all,
```

Comment:
`sum` seems off since it's more like a count of migrations - I think I'll leave as-is and assume `quantity` covers it (and also helps users link back to the implementation probes)